src/index.md

```diff
@@ -37,7 +37,7 @@ Install our command-line interface:
 $ npm install workbox-cli --global
 
 # Generate a service worker with some smart defaults
-$ workbox generate:sw
+$ workbox-cli generate:sw
 ```
```

Comment:
Sorry for mentioning this in passing this late on @gauntface and @jeffposnick. Is there a reason we would prefer the more verbose `workbox-cli` instead of just `workbox` here? (I must have missed this while we were reviewing earlier). Won't hold back this review on the question at all.